tests/sqlparser_postgres.rs

```diff
@@ -401,6 +401,21 @@ PHP	₱ USD $
     //assert_eq!(sql, ast.to_string());
 }
 
+#[test]
+fn parse_update_set_from() {
+    let sql = "UPDATE t1
+    SET name = t2.name FROM ( \
+        SELECT name, id \
+        FROM t1 \
+        GROUP BY id \
+    ) t2 \
+    WHERE t1.id = t2.id";
+    let ast = pg().one_statement_parses_to(sql, "");
+    println!("{:#?}", ast);
+    println!("{}", ast);
+    //assert_eq!(sql, ast.to_string());
```

Comment:
It seems like this test is missing the expected ast. Can you please update that?

Reply:
Added, not sure whether I'm doing right 😄.